pkgs/development/libraries/gstreamer/good/default.nix

```
@@ -190,6 +190,6 @@ stdenv.mkDerivation rec {
     '';
     license = licenses.lgpl2Plus;
     platforms = platforms.linux ++ platforms.darwin;
```

```suggestion
    platforms = platforms.unix;
```
most others are unix. maybe a separate cleanup PR would be a good idea

---

If you do one or two or three smaller PRs I can review and merge them faster since there are probably not so many changes you can talk about.